### DISCUSSION TITLE
Is there a way to quickly toggle checking?

### DISCUSSION THREAD

**Discussion by TahaAhmedRasheed** [davidlday/vscode-languagetool-linter#763] [Q&A]
Is there an option within the extension to turn off checking? So far I've had to disable the extension altogether which also then requires restarting all extensions in VSCode and so does take up some time. The `"languageToolLinter.enabled"` setting has ostensibly no effect. 

My usecase is to only perform grammar checking on-demand.

**Comment by davidlday** [comment]
Hey! That's a great question. The short answer is no.

At one point, I had a feature that kind of did this, but it's no longer working right. There's an issue open asking for the same here: https://github.com/davidlday/vscode-languagetool-linter/issues/750

I plan on implementing this appropriately soon. I'm also looking into adding a status bar icon to show if the linter is enabled and give quick access to toggling linting on demand.
